test/cmdlineTests/stdin/stdin

```diff
@@ -0,0 +1 @@
+contract C {}
```

Comment:
IIUC, to use `stdin` a `-` is passed to solc cmdline and a file called `stdin` is created to simulate standard input?

Reply:
Yeah. You create the `stdin` file when creating the test (if you want to use it).